test/unit/Command.orderbook.spec.ts

```diff
@@ -0,0 +1,109 @@
+import uuidv1 from 'uuid';
+import assert from 'assert';
+import { expect } from 'chai';
+import { Order } from '../../lib/proto/xudrpc_pb';
+import { createOrderbookSide } from '../../lib/cli/commands/orderbook';
+import { performance } from 'perf_hooks';
+
+const createOrders = (
+  amount: number,
+  price: number,
+  quantity: number,
+  randomAmounts = false,
+): Order.AsObject[] => {
+  assert(amount >= 1, 'amount must greater than 0');
+  const randomNumber = () => {
+    return Math.round(Math.random() * (Number.MAX_SAFE_INTEGER - 1) + 1);
+  };
+  return Array.from(Array(amount))
+    .map(() => {
+      const id = uuidv1();
+      return {
+        id,
+        price: randomAmounts ? randomNumber() : price,
+        quantity: randomAmounts ? randomNumber() : quantity,
+        localId: id,
+        pairId: 'LTC/BTC',
+        peerPubKey: '123',
+        createdAt: Date.now(),
+        side: 0,
+        isOwnOrder: false,
+        hold: 0,
+      };
+    });
+};
+
+describe('Command.orderbook.createOrderbookSide', () => {
+  it('generates orderbook from 1 million orders', () => {
+    const orders: Order.AsObject[] = createOrders(1000000, 0.012160, 100000, true);
+    const startTime = performance.now();
+    createOrderbookSide(orders);
+    const endTime = performance.now();
+    const timeSpent = endTime - startTime;
+    expect(timeSpent < 6000).to.equal(true);
```

Comment:
Think we can lower this test to check for 10k instead of 1M orders in order to not slow down the test suite too much unless someone sees a compelling reason not to do so.

Reply:
I agree, in fact I'm not sure we need to have performance tests here in general since it's so hardware dependent. Tests that pass for us might fail on Travis because of differences in available resources.

Reply:
I opted to keep the test for future debugging purposes to have an easy way to benchmark, but reduced the amount of orders and increased the buffer time so that I'll pass even on slow machines. The alternative is to skip the test and enable it when needed. Actually, that sounds better. I'll make the change.

Reply:
Done - skipped that test.